knative-operator/hack/certnames.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/knative-operator/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/internal/cert/writer/certwriter.go b/knative-operator/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/internal/cert/writer/certwriter.go
+index ed6b5110..c0c67e16 100644
+--- a/knative-operator/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/internal/cert/writer/certwriter.go
++++ b/knative-operator/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/internal/cert/writer/certwriter.go
+@@ -33,9 +33,9 @@ const (
+ 	// CACertName is the name of the CA certificate
+ 	CACertName = "ca-cert.pem"
+ 	// ServerKeyName is the name of the server private key
+-	ServerKeyName = "key.pem"
++	ServerKeyName = "apiserver.key"
+ 	// ServerCertName is the name of the serving certificate
+-	ServerCertName = "cert.pem"
++	ServerCertName = "apiserver.crt"
```

Comment:
Just a question: Are there any plans to make the name of the cert's location configurable in the upstream controller runtime ? sure, it's fixed for Kubernetes but other distributions like Rancher might have similar issues. On the other hand, I wonder why OpenShift changed that naming ....

Reply:
Yes, this is fixed in a newer c-r version. The followup PR for this (that'll bump the version) will have this configurable in client code rather than patching vendor.